getting-started-tutorial/tutorial-2-Model-training.ipynb

```diff
@@ -0,0 +1,2021 @@
+{
```

Comment:
<p>it actually uses:</p><p><code>get_data_run</code>  =&gt; <code>train_run</code>  and it's on <code>step 1</code>  </p><p><code>get_data</code>  is maybe from the previous tutorial but not this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>